tests/AppIntegrationTest.php

```diff
@@ -235,6 +235,22 @@ static function ($request) {
         $this->assertSame('second', $this->client->get('/')->getBody()->getContents());
     }
 
+    public function testServerLogsAreNotInErrorOutput()
+    {
+        $this->client->delete('/_all');
+
+        $expectedServerErrorOutput = '[404]: (null) / - No such file or directory';
+
+        self::$server1->addErrorOutput('PHP 7.4.2 Development Server (http://localhost:8086) started' . PHP_EOL);
+        self::$server1->addErrorOutput('Accepted' . PHP_EOL);
+        self::$server1->addErrorOutput($expectedServerErrorOutput . PHP_EOL);
```

Comment:
I'm not sure, but I think only this line would be enough to test.

Reply:
I think it's good to have an entire log of a real case to test, so we test both cases, the lines that has to be erased and the one that has to stay.